ui/src/tempVars/components/TemplateDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {SFC} from 'react'
+
+import Dropdown from 'src/shared/components/Dropdown'
+
+import {Template, TemplateValue, TemplateValueType} from 'src/types'
+
+interface Props {
+  template: Template
+  onPickValue: (v: TemplateValue) => void
+}
+
+const TemplateDropdown: SFC<Props> = props => {
+  const {template, onPickValue} = props
+
+  const dropdownItems = template.values.map(value => {
```

Comment:
how do you feel about using a `get` for template values in the case that there are no values?